### ISSUE TITLE
Trying to upload artboard

### ISSUE THREAD

**Issue by andersLandau** [avocode/avocode#1523]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.14.6`
Avocode UID: 141424
```
Error: ENOENT: no such file or directory, stat '/var/folders/9d/78cdqvp93kb7w7f60h873j2r0000gn/T/com.madebysource.avocode/avocode-sync-1499947954821/ABA45AAA-E211-4DC2-82E9-19DE073CA6D1/A2D409AE-E49E-4903-9348-734852F7F36A.png'
```

**Comment by jakubzitny** [comment]
Hi @andersLandau, thanks for the report. This looks like #1511 which is supposed to be fixed in `2.14.6`. Let me look into it. Did it happen more than once to you? Can you send us the design to team@avocode.com, please, so we can test it more thoroughly?

**Comment by connor-bean** [comment]
Hey @andersLandau -

Would you be able to send us this file by chance?

**Comment by andersLandau** [comment]
Here you go

https://www.dropbox.com/s/1oranmnx8gqo018/Snaplytics%20-%20App%20-%20Design%20-%205.zip?dl=0

--
Anders Landau
CPO/Co-Founder

P: +45 3135 0600
E: anders@snaplytics.io <mailto:thomas@snaplytics.io>
W: snaplytics.io <http://snaplytics.io/>

**Comment by jakubzitny** [comment]
Thanks, I'll look into it.

**Comment by jakubzitny** [comment]
Hi there, this is fixed in latest Avocode `2.15.0`. You can download it from [avocode.com/download](https://avocode.com/download) or use auto-update to install it. Please give it a try and let us know if you have any more issues.
